### PR TITLE
HELPS-1032: making selfTestLoop run on windows

### PR DESCRIPTION
**Pull request by RusGraf**

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/config/whisper-validation-group.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const whisperValidationTestGroup = (): TestGroup =>
     new LoopTest(
       'Whisper Validation - Basic Form Validation',
       whisperValidationTests.testComponentsValidation,
-      20000,
+      30000,
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/config/whisper-validation-group.ts:11]: Thanks - this was hard to do in time.

File: ldk/javascript/examples/self-test-loop/src/utils/string.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export enum StringOptions {
+  IgnoreCase,
+}
+
+// TODO: Required as windows filesystem create converts file path to lower case. Remove after sidekick is fixed. Ian is working on it.
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/utils/string.ts:5]: Is there a Jira card number? This comment is probably more relevant on the test that invokes `areStringsEqual`

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/examples/self-test-loop/src/config/whisper-validation-group.ts (hunk 1 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/config/whisper-validation-group.ts:11]: lol, right

**Review by epiccoleman — APPROVED** [review]
Looks good - nice refactors.

File: ldk/javascript/examples/self-test-loop/src/utils/string.ts (hunk 2 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/utils/string.ts]: Updated.

**Pull request merged**